src/cleanvision/imagelab.py

```diff
@@ -52,6 +58,15 @@ class Imagelab:
         Issue checks will be run on this list of image paths specified in `filepaths`.
         Specifying only one of `data_path` or `filepaths`.
```

Comment:
This phrase is outdated. It should say:

Specify only one of: <list all args>

Reply:
Updated